lobby-db-dao/src/main/java/org/triplea/lobby/server/db/data/UserRole.java

```diff
@@ -0,0 +1,16 @@
+package org.triplea.lobby.server.db.data;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+/**
+ * Class to hold constants representing the possible set of user role values. <br>
+ * Note: this is not an enum to allow these values to be referenced from annotations.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class UserRole {
+  public static final String ADMIN = "ADMIN";
+  public static final String MODERATOR = "MODERATOR";
+  public static final String PLAYER = "PLAYER";
+  public static final String ANONYMOUS = "ANONYMOUS";
```

Comment:
What about using an enum here?
I believe somewhere else in the code we should have a similar enum (ADMIN, NOT_ADMIN IIRC)

Reply:
An enum makes sense, but it takes a lot of hackery to then use strings in annotations. My first approach was an enum, but then you wind up creating a static inner class to hold the string values, and you map the enum constants to strings, and then you wind up needing to override @ToString and then there is confusion whether 'name()' or 'toString' should be used.

Seemed easier in the end to just use string constants. There is a comment in javadoc to indicate an enum was considered: ` Note: this is not an enum to allow these values to be referenced from annotations.`